source/hiring/index.md

```diff
@@ -0,0 +1,88 @@
+# arXiv is hiring!
+
+We're looking for a community engagement manager, a few good developers, a program manager, a tech writer/documentation specialist and more.
+
+ - arXiv is part of Cornell University. All arXiv employees are Cornell University employees.
+ - The positions listed on this page are all full time [staff positions with benefits (endowed)](https://hr.cornell.edu/understand-your-benefits).
+ - Visa sponsorship is not available for these positions.
+ - Unless otherwise stated, these are 3-year renewable appointments
+ - arXiv is headquartered at the [Cornell Tech campus on Roosevelt Island in New York City](https://tech.cornell.edu/). There is a strong preference for candidates who can report to work at that location, but there are options for hybrid remote work.
+
+We are presenlty looking for [Software Developers](#software-developers), a [Project Manager](#project-manager), and a [Community Engagement Manager](#community-engagement-manager).
```

Comment:
Possibly bullet this list so they pop out more.

Reply:
They're well-separated below. This is deliberately a short list that doesn't take up too much space.